src/Umbraco.Core/Models/ContentVariation.cs

```diff
@@ -3,34 +3,35 @@
 namespace Umbraco.Core.Models
 {
     /// <summary>
-    /// Indicates the values accepted by a property.
+    /// Indicates how values can vary.
     /// </summary>
+    /// <remarks>
+    /// <para>Values can vary by nothing, or culture, or segment, or both.</para>
+    /// <para>Varying by culture implies that each culture version of a document can
+    /// be available or not, and published or not, individually. Varying by segment
+    /// is a property-level thing.</para>
+    /// </remarks>
     [Flags]
     public enum ContentVariation : byte
     {
         /// <summary>
-        /// Unknown.
+        /// Values do not vary.
         /// </summary>
-        Unknown = 0,
+        Nothing = 0,
 
         /// <summary>
-        /// Accepts values for the invariant culture and the neutral segment.
+        /// Values vary by culture.
         /// </summary>
-        InvariantNeutral = 1,
+        Culture = 1,
 
         /// <summary>
-        /// Accepts values for a specified culture and the neutral segment.
+        /// Values vary by segment.
         /// </summary>
-        CultureNeutral = 2,
+        Segment = 2,
 
         /// <summary>
-        /// Accepts values for the invariant culture and a specified segment.
+        /// Values vary by culture and segment.
         /// </summary>
-        InvariantSegment = 4,
-
-        /// <summary>
-        /// Accepts values for a specified culture and a specified segment.
-        /// </summary>
-        CultureSegment = 8
+        CultureAndSegment = Culture | Segment
```

Comment:
Ah, i see why flags is still there then ... but does that make sense since it's still illegal to have `Nothing | CultureAndSegment` right?

Reply:
`Nothing` is zero, so `Nothing|CultureAndSegment` is same as `CultureAndSegment` is same as `Culture|Segment` - going to remove the flag thing